src/utils/flag.js

```diff
@@ -0,0 +1,169 @@
+import { Dimensions } from "react-native";
+import React from "react";
+import { Modal, SafeAreaView, TouchableOpacity, View, Text, TextInput } from "react-native";
+import Flag from "react-native-flags";
+import { Button } from "../components/general/";
+
+const dimension = Dimensions.get("window");
+
+export const RenderFlags = (props) => {
```

Comment:
I also commented about this earlier. This is also good just delete the flag stuff from dashboard